frontend-instructor/src/components/courseManagement/CourseForm.jsx

```
@@ -0,0 +1,201 @@
+        payload
+      );
+
+      if (result.success) {
+        if (isEdit) {
+          alert('Course updated successfully');
+        } else {
+          resetForm();
+          if (onSuccess) onSuccess(result);
+        }
+      } else {
+        alert('Error: ' + (result.message || 'Could not save course'));
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)